src/frontend/scss/objects/_form.scss

```diff
@@ -25,6 +25,14 @@
       margin-bottom: 0;
     }
 
+    &-button-container {
+      // Cunningham buttons are 8px smaller than inputs
+      // to align them, we need to add this small margin top.
+      // row's align-item: center doesn't work because when an input
+      // is on error, it can add multiple line underneath
+      margin-top: rem-calc(4px);
+    }
+
```

Comment:
The margin-top looks not applied. Furthermore, why not simply center elements through a flexbox ?
<img width="205" alt="image" src="https://github.com/openfun/richie/assets/9265241/70149a20-869f-4266-a788-4bf4bbbd7311">

Reply:
Weird, i have it well centered on my storybook
![image](https://github.com/openfun/richie/assets/139848/8db04992-dae0-4dfa-b91c-7002154f3a66)

did you rebuild css files ?

flex didn't work because when we display the error line, it center on the input plus the error line.

Reply:
Sure I think you're right, I forgot to rebuild css.